### PR TITLE
New version: OpenRCT2.OpenRCT2 version 0.4.0

### PR DESCRIPTION
**Pull request by Broxzier**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/58768)

### CODE REVIEW

File: manifests/o/OpenRCT2/OpenRCT2/0.4.0/OpenRCT2.OpenRCT2.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: OpenRCT2.OpenRCT2
+PackageVersion: 0.4.0
```
Comment (OfficialEsco) [inline manifests/o/OpenRCT2/OpenRCT2/0.4.0/OpenRCT2.OpenRCT2.installer.yaml:5]:
```suggestion
PackageVersion: 0.4.0master-c6302a1
```

File: manifests/o/OpenRCT2/OpenRCT2/0.4.0/OpenRCT2.OpenRCT2.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: OpenRCT2.OpenRCT2
+PackageVersion: 0.4.0
```
Comment (OfficialEsco) [inline manifests/o/OpenRCT2/OpenRCT2/0.4.0/OpenRCT2.OpenRCT2.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.4.0master-c6302a1
```

File: manifests/o/OpenRCT2/OpenRCT2/0.4.0/OpenRCT2.OpenRCT2.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: OpenRCT2.OpenRCT2
+PackageVersion: 0.4.0
```
Comment (OfficialEsco) [inline manifests/o/OpenRCT2/OpenRCT2/0.4.0/OpenRCT2.OpenRCT2.yaml:5]:
```suggestion
PackageVersion: 0.4.0master-c6302a1
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Someone messed with the DisplayVersion in the 0.4.0 release...
```
Enter a pull request number: 58768

Manifest validation succeeded.
Found OpenRCT2 [OpenRCT2.OpenRCT2] Version 0.4.0
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/OpenRCT2/OpenRCT2/releases/download/v0.4.0/OpenRCT2-0.4.0-windows-installer-x64.exe
  ██████████████████████████████  18.8 MB / 18.8 MB
Successfully verified installer hash
Starting package install...
Successfully installed

Name          : OpenRCT2 0.4.0master-c6302a1
Publisher     : OpenRCT2
Version       : 0.4.0master-c6302a1
ProductCode   : OpenRCT2
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\OpenRCT2
Uninstall     : winget uninstall --id "OpenRCT2"
```

File: manifests/o/OpenRCT2/OpenRCT2/0.4.0/OpenRCT2.OpenRCT2.installer.yaml (hunk 1 above)
Comment (Broxzier) [reply manifests/o/OpenRCT2/OpenRCT2/0.4.0/OpenRCT2.OpenRCT2.installer.yaml:5]: Makes sense that this returned, this was also the case in version `0.3.4master-e0daac9`, but versioning got broken after that release and has been fixed with this one. I expected the `winget validate` or the `SandboxTest.ps1` script to catch this.

Comment (OfficialEsco) [reply manifests/o/OpenRCT2/OpenRCT2/0.4.0/OpenRCT2.OpenRCT2.installer.yaml:5]: winget validate just validates that the manifest is in the correct format.
SandboxTest.ps1 can be used to install the software and check the Add or Remove Programs metadata fields so its a manual process.
In the future wingetbot will look for incorrect metadata, but thats... the future 😁

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**